website/docs/changelog.md

```diff
@@ -0,0 +1,55 @@
+# Changelog
+
+## v2.0 - Mar 28, 2025
+
+This release introduces several new controllers, expanding ORC's capabilities
+beyond the original image controller. With this update, ORC now offers
+a robust, stable core and a comprehensive end-to-end (e2e) test suite, making
+it easier to create new controllers while ensuring quality and reliability.
+
+Version 2.0 highlights the capabilities of ORC and the direction the project
+wants to take. The API is still alpha and may change frequently.
+
+### New controllers
+
+- Flavor
+- Network
+- Port
+- Router
+- Security Group
+- Server
+- Subnet
+
+### Breaking changes
+
+```
+github.com/k-orc/openstack-resource-controller/api/v1alpha1
+  Incompatible changes:
+  - ImageFilter.Name: changed from *string to *OpenStackName
+  - ImageFilter: old is comparable, new is not
```

Comment:
What does 'comparable' mean in this context?

Reply:
Ah, ok. That's not the level I would have chosen to describe this change, but I can see that it make sense.